### PR TITLE
feat: Update android docs

### PR DESCRIPTION
**Pull request by matgar26**
### Description

Updating Android documentation based on new installation method

### CODE REVIEW

**Comment by thomaswhyyou** [comment]
@matgar26 Once we merge this PR, we can probably close #375? (cc: @cellomatt)

File: content/in-app-ui/android/quick-start.mdx [hunk 1]
```diff
@@ -8,40 +8,141 @@ The Knock Kotlin SDK is a client-side SDK for interacting with the Knock API and
...
+      </td>
+    </tr>
+    <tr width="600px">
+      <td align="left">Minimum Android SDK Version</td>
+      <td align="center">
+        <code>29</code>
+      </td>
+    </tr>
+  </tbody>
+</table>
```
Comment (cjbell) [inline content/in-app-ui/android/quick-start.mdx:43]: Here can we use markdown syntax for this please: https://www.markdownguide.org/extended-syntax/#tables

**Review by cjbell — COMMENTED** [review]
one comment!

File: content/in-app-ui/android/quick-start.mdx (hunk 1 above)
Comment (matgar26) [reply content/in-app-ui/android/quick-start.mdx]: @cjbell This is now updated. Thanks for the reference!

**Review by cjbell — APPROVED** [review]
💯

**Pull request merged**